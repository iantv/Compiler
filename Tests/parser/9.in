void a();
void a();

void a(){ 1; };
void a();