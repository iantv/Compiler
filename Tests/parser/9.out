a: function
	expression:
		1
return
	void
